SonyPSPResolveNIDs.py

```diff
@@ -296,13 +297,13 @@ def getModuleInfoAddrFromLoadCommands():
     # 2nd component is p_offset
     load_offset = loadcmd.getComponent(1).getValue().getValue() # Data->Scalar->Long
```

Comment:
What do you think about using 
`load_offset = loadcmd.getComponent(1).getLong(0)`?

The same can be applied to the 299th line as well (make it `load_paddr = loadcmd.getComponent(3).getLong(0)`), I think.

Reply:
That seems to work, looks better too.